### ISSUE TITLE
ToolbarPrimitive.ToggleItem.onClick fires twice

### ISSUE THREAD

**Issue by wallynm** [radix-ui/primitives#1525]
Bug report
Current Behavior
Check console in sandbox.

https://codesandbox.io/s/frosty-kapitsa-3b6hbr?file=/App.js

Expected behavior
onClick only fires once

Similar issue related with: https://github.com/radix-ui/primitives/issues/1145

**Comment by bukinoshita** [comment]
I think you need to use the `onValueChange` on the `ToolbarToggleGroup` instead of using the `onClick` on the `ToolbarToggleItem`.

Eg: (and here's a [CodeSandbox](https://codesandbox.io/s/kind-snyder-o4ik44?file=/App.js))

```tsx
<ToolbarToggleGroup
  type="single"
  aria-label="Text formatting"
  onValueChange={(value) => console.log(value)}
>
  <ToolbarToggleItem value="bold" aria-label="Bold">
    <FontBoldIcon />
  </ToolbarToggleItem>
  <ToolbarToggleItem value="italic" aria-label="Italic">
    <FontItalicIcon />
  </ToolbarToggleItem>
  <ToolbarToggleItem value="strikethrough" aria-label="Strike through">
    <StrikethroughIcon />
  </ToolbarToggleItem>
</ToolbarToggleGroup>
```

**Comment by benoitgrelard** [comment]
Yeah that would be the correct way to use the component most of the time, that being said, it's still a bug in case someone was to attach an `onClick`, let's say for analytics reasons for example.

**Comment by bukinoshita** [comment]
@benoitgrelard I have worked on a PR that fixes that which uses a similar approach on https://github.com/radix-ui/primitives/issues/1145, but wasn't sure since it worth opening since it's we should be using the `onValueChange` prop instead. If this is something that we still want to fix, I can open that PR for review

**Comment by wallynm** [comment]
I will finish the patch today, so we can merge

**Comment by benoitgrelard** [comment]
Fixed by #1526
